Rocket.Chat/API/APIRequest.swift

```diff
@@ -58,7 +58,7 @@ extension APIRequest {
 
     func request(for api: API, options: APIRequestOptions = .none) -> URLRequest? {
         var components = URLComponents(url: api.host, resolvingAgainstBaseURL: false)
-        components?.path = path
+        components?.path += path
```

Comment:
@RocketChat/ios Guys, this `+=` has actually introduced a bug, because we specify our paths with as the following: `/api/v1/me`, and if we concatenate this path with the default path of a `URLComponents` it will result in a duplicated initial slash as the following: `//api/v1/me` which makes the request invalid. 

It took me a while to discover what was going on because the result it is generating is not even handled as an error by the app, so it generated confusing behaviors such as making the app stuck on an endless loading in the auth screen, or even authenticating the user with a `nil` user 🕵️

Reply:
#1415 fixes it

Reply:
@filipealva Thank you! 👍